### PR TITLE
Also update ec-task-policy image ref

### PR DESCRIPTION
**Pull request by lcarva**
The infra-deployments repo now contains ECP resources that use the task policy bundle. This commit teaches the update script to handle updating those references as well.

Ref: EC-358

### CODE REVIEW

File: hack/update-infra-deployments.sh [hunk 1]
```diff
@@ -53,24 +54,31 @@ function oci_source() {
...
       yq e -e \
           '(select(has("kind")) | select(.kind == "EnterpriseContractPolicy"))' \
-          $yaml_file || continue
-      # Finally, update the source references.
-      SOURCE_KEY="${1}" SOURCE_URL="${2}" yq e -i \
-          '.spec.sources[0][env(SOURCE_KEY)][0] |= env(SOURCE_URL) | .' \
-          $yaml_file
+          $yaml_file 2> /dev/null || continue
+      # Finally, update the source references. A previous iteration used yq to perform a more
+      # precise update. However, making a conditional update is non-trivial. sed is a simpler here.
```
Comment (robnester-rh) [inline hack/update-infra-deployments.sh:68]: nit: "sed is a simpler option here." or "sed is simpler."

Comment (robnester-rh) [inline hack/update-infra-deployments.sh:68]: nit: "sed is a simpler option here." or "sed is simpler."

**Review by robnester-rh — APPROVED** [review]
Looks good, only nit was wording on comments which doesn't affect functionality.

File: hack/update-infra-deployments.sh (hunk 1 above)
Comment (lcarva) [reply hack/update-infra-deployments.sh]: Thanks! Fixed it.

**Pull request merged**